tools/smoke-tests/model-compiler-tests/build.gradle.kts

```diff
@@ -18,10 +18,7 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-import io.spine.gradle.internal.Deps
-import io.spine.tools.protoc.MessageSelectorFactory.prefix
-import io.spine.tools.protoc.MessageSelectorFactory.regex
-import io.spine.tools.protoc.MessageSelectorFactory.suffix
+import io.spine.tools.protoc.MessageSelectorFactory.*
 
```

Comment:
Can we avoid start imports in Kotlin too, please?

Reply:
Definetely. Will update with a proper change to the `config` (see https://github.com/SpineEventEngine/config/pull/161).